src/aiohubspace/v1/controllers/event.py

```diff
@@ -182,7 +183,11 @@ async def __event_reader(self) -> None:
                         )
                     )
                     processed_ids.append(hs_dev.id)
-            except (ClientError, asyncio.TimeoutError) as err:  # pragma: no cover
+            except (
+                ClientError,
+                asyncio.TimeoutError,
+                HTTPForbidden,
+            ) as err:  # pragma: no cover

```

Comment:
`# pragma: no cover` should probably be removed from here and the othe exceptions. They should be included in coverage.